.github/workflows/e2e.yml

```diff
@@ -40,10 +40,10 @@ jobs:
       -
         name: Update daemon.json
         run: |
-          sudo jq '.experimental = true' < /etc/docker/daemon.json > /tmp/docker.json
-          sudo mv /tmp/docker.json /etc/docker/daemon.json
-          sudo cat /etc/docker/daemon.json
-          sudo service docker restart
+          # sudo jq '.experimental = true' < /etc/docker/daemon.json > /tmp/docker.json
+          # sudo mv /tmp/docker.json /etc/docker/daemon.json
+          # sudo cat /etc/docker/daemon.json
+          # sudo service docker restart
```

Comment:
I notice that this is executed _both_ for `experimental` and `non-experimental` - curious now if that's wrong, or if we just enabled experimental to allow tests that use it to run.

Not critical, but I guess we could skip this step to not having to restart the daemon for the non-experimental ones.